pkg/commands/run.go

```diff
@@ -131,6 +131,10 @@ func initFlagSet(fs *pflag.FlagSet, cfg *config.Config, m *lintersdb.Manager, is
 		30, "Minimal complexity of function to report it")
 	hideFlag("gocyclo.min-complexity")
 
+	fs.IntVar(&lsc.Gocognit.MinComplexity, "gocognit.min-complexity",
```

Comment:
don't add a command-line flag, please. It's the deprecated way of configuration.

Reply:
I see.. Changes applied